apps/database/supabase/tests/private-schema-external-chat-parity.sql

```
@@ -0,0 +1,156 @@
+      and table_name in (
+        'external_chat_source_events', 'external_chat_observations',
+        'external_chat_sync_runs', 'external_chat_stream_cursors'
+      )
+      and grantee in ('anon', 'authenticated')$$,
+  'parity stores have no direct client grants'
+);
+
+create temporary table parity_context (ws_id uuid primary key, actor_id uuid not null);
+insert into parity_context
+select '77777777-7777-4777-8777-777777777777'::uuid, id
+from public.users order by created_at limit 1;
```

SQLFluff (4.2.2)

[error] The 'order' keyword should always start a new line.

(LT14)